### ISSUE TITLE
G0 - Markup Validation Errors

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#127]
There are several errors generated in the markup validator that need to be fixed to pass 4.1.1 Parsing on the WCAG 2.0: https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2F

**Comment by jpinsent** [comment]
@mmafe Here's a list of all the validator reports for the pages we tested:

[Front Page](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2F)

[Basic Page (Image Guide)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fgryph-mail-zimbra-8%2Fweb-client-overview%2Fweb-version-email%2Fgryph-mail-basics)

[Basic Page (Grid)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fgryph-mail-zimbra-8%2Fweb-client-overview%2Fmobile-version-setup)

[Basic Page (Table)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fnode%2F859)

[Events Listing](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fccs-service-updates)

[News Listing (All news)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fnews)

[News Listing(Filtered by Date)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fnews%2Farchive%2F201208)

[FAQ Listing (All)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Ffaq)

[FAQ Listing (Filtered by Category)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Ffaq%2Fcategory%2F89)

[Service Offering](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fservice%2Fvoice-mail)

[Service Listing (Category)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fservices)

[Service Listing (Alphabetical)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fservices%2Falphabetical)

[Service Listing (Audience)](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fservices%2Faudience)

[News Article](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fnews%2Fmanaged-servers)

[Event/Service Update](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fccs-service-updates%2Femergency-maintenance-internet-edge-router-january-13th-500-600pm)

[FAQ Question](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fwhat-contributes-my-quota)

[Random Basic Page](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fnode%2F785)

[Random News Item](https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.uoguelph.ca%2Fccs%2Fnode%2F1059)

**Comment by mmafe** [comment]
Thanks Josh!

The following has been fixed on dev. I'll hold off on closing until we've pushed the changes live.

**Fixed**
- Front Page
- Basic Page (Image Guide)
- Basic Page (Grid)
- Events Listing
- FAQ Listing (Filtered by Category)
- Service Offering
- Service Listing (Category)
- Service Listing (Alphabetical)
- Service Listing (Audience)
- Event/Service Update
- FAQ Question
- Random Basic Page

**Needs to be fixed in content**
- Basic Page (Table)
- FAQ Listing (All)

**Out of Scope - News Subscribe Widget has validation errors**
- News Listing (All news)
- News Listing(Filtered by Date)
- News Article
- Random News Item

**Comment by jpinsent** [comment]
Content validation errors in Basic Page (Table) and FAQ Listing (All) have been fixed on the live site. There may be a few more issues with tables on other pages, but Siteimprove should be able to pick those up.

**Comment by mmafe** [comment]
Fixed. Already pushed to dev master (prior to Hotfix branch creation).

**Comment by jszasz** [comment]
Confirmed, Fixed.
